src/monmove.cpp

```diff
@@ -65,6 +65,7 @@ static const efftype_id effect_dragging( "dragging" );
 static const efftype_id effect_grabbed( "grabbed" );
 static const efftype_id effect_harnessed( "harnessed" );
 static const efftype_id effect_led_by_leash( "led_by_leash" );
+static const efftype_id effect_immobilization( "immobilization" );
```

Comment:
```suggestion
static const efftype_id effect_immobilization( "immobilization" );
static const efftype_id effect_led_by_leash( "led_by_leash" );
```

Oops OK, github's suggestion UI is finnicky. Let's try this.